### ISSUE TITLE
Andorid - Can't play same song twice in a row

### ISSUE THREAD

**Issue by jmartine2** [martijn00/XamarinMediaManager#71]
I'm unable to play the same song twice in a row. I traced the code and here's what's happening:

The first time the song is played, it plays fine.
The second time, MediaServiceBase.Play() calls CheckIfFileAlreadyIsPlaying(), which in this situation sets the song position to zero and returns True, which causes the Play() function to return without doing anything. 

CheckIfFileIsAlreadyPlaying() is returning True because of the following check: (mediaFile?.Url == CurrentFile?.Url) which I guess will be True in this case because CurrentFile still contains the same Url as the last time it was played.

Any thoughts on the best way to fix this?

**Comment by bubavanhalen** [comment]
i did notice it too, 

it should be fixed with this pull request:

https://github.com/martijn00/XamarinMediaManager/pull/70/files#diff-ca33a496d9995f3f564402023abd0230

**Comment by jmartine2** [comment]
Great, thanks!

**Comment by jmartine2** [comment]
I just tested it, and this issue is still not fixed. Although some of the logic was changed in CheckIfFileAlreadyIsPlaying() in the referenced pull request, it still returns True in the case where the same song is played again, from the following piece of code:

            //Same file selected => seek to start
            if (MediaPlayerState != PlaybackStateCompat.StatePaused && !ManuallyPaused)
            {
                await Seek(TimeSpan.Zero);
                return await Task.FromResult(true);
            }

And therefore Play() returns without playing the song.

This issue should be re-opened. I'd be happy to fix it, but I need some input as to the best way.
